builtin/bisect--helper.c

```
@@ -195,14 +196,64 @@ static int check_expected_revs(const char **revs, int rev_nr)
 	return 0;
 }
 
+static int bisect_write(const char *state, const char *rev,
+			const char *term_good, const char *term_bad,
+			int nolog)
+{
+	struct strbuf tag = STRBUF_INIT;
+	struct strbuf commit_name = STRBUF_INIT;
+	struct object_id oid;
+	struct commit *commit;
+	struct pretty_print_context pp = {0};
+	FILE *fp;
+
+	if (!strcmp(term_bad, state))
+		strbuf_addf(&tag, "refs/bisect/%s", state);
+	else if(one_of(state, term_good, "skip", NULL))
+		strbuf_addf(&tag, "refs/bisect/%s-%s", state, rev);
+	else
+		return error(_("Bad bisect_write argument: %s"), state);
+
+	if (get_oid(rev, &oid)) {
+		strbuf_release(&tag);
+		return -1;
```

I wanted to ask about this kind of stuff. In a lot of cases the shell script doesn't show the error message when error occurs but silently returns -1. Should I from henceforth see to the fact that error message are printed whenever possible?